source/stylesheets/refills/unstyled/_navigation-centered.scss

```diff
@@ -0,0 +1,93 @@
+.navigation-centered {
+  $large-screen: em(860) !default;
+  $centered-navigation-height: 60px;
+  $centered-navigation-logo-height: 2em;
+
+  width: 100%;
+
+  // Mobile view
+  .mobile-logo {
+    display: inline;
+    float: left;
+
+    img {
+      max-height: $centered-navigation-logo-height;
+    }
+
+    @include media($large-screen) {
+      display: none;
+    }
+  }
+
+  .navigation-centered-mobile-menu {
+    float: right;
+    padding-top: 0.5em;
+
+    @include media ($large-screen) {
+      display: none;
+    }
+  }
+
+  // Nav menu
+  .navigation-centered-wrapper {
+    @include clearfix;
+    @include outer-container;
+    position: relative;
+  }
+
+  ul.navigation-centered-menu {
+    clear: both;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets